tests/cluster-users-test/sar-test.sh

```diff
@@ -3,83 +3,87 @@
+	readonly RESPONSE=$(curl -X GET "${DEX_SERVICE_SERVICE_HOST}:${DEX_SERVICE_SERVICE_PORT_HTTP}/approval?${REQUEST_ID}")
+	readonly AUTH_TOKEN=$(echo "${RESPONSE}" | grep -o -P '(?<=id_token=).*(?=&amp;state)')
+	curl -s -H "Authorization: Bearer ${AUTH_TOKEN}" "${CONFIGURATIONS_GENERATOR_SERVICE_HOST}:${CONFIGURATIONS_GENERATOR_SERVICE_PORT_HTTP}/kube-config" -o "${PWD}/kubeconfig"
+}
 
-	echo "--> developer@kyma.cx should be able to create Deployments in ${NAMESPACE}"
-	testPermissions "developer@kyma.cx" "create" "deploy" "${NAMESPACE}" "yes"
+function runTests() {
+    EMAIL=${DEVELOPER_EMAIL} PASSWORD=${DEVELOPER_PASSWORD} getConfigFile
+    export KUBECONFIG="${PWD}/kubeconfig"
+	echo "--> ${DEVELOPER_EMAIL} should be able to get Deployments in ${NAMESPACE}"
+	testPermissions "get" "deploy" "${NAMESPACE}" "yes"
 
-	echo "--> developer@kyma.cx should be able to get CRD in ${NAMESPACE}"
-	testPermissions "developer@kyma.cx" "get" "crd" "${NAMESPACE}" "yes"
+	echo "--> ${DEVELOPER_EMAIL} should be able to create Deployments in ${NAMESPACE}"
+	testPermissions "create" "deploy" "${NAMESPACE}" "yes"
 
-	echo "--> developer@kyma.cx should be able to get specific CRD in ${NAMESPACE}"
-	testPermissions "developer@kyma.cx" "get" "crd/installations.installer.kyma-project.io" "${NAMESPACE}" "yes"
+	echo "--> ${DEVELOPER_EMAIL} should be able to get CRD in ${NAMESPACE}"
+	testPermissions "get" "crd" "${NAMESPACE}" "yes"
 
-	echo "--> developer@kyma.cx should NOT be able to delete ClusterRole in ${NAMESPACE}"
-	testPermissions "developer@kyma.cx" "delete" "clusterrole" "${NAMESPACE}" "no"
+	echo "--> ${DEVELOPER_EMAIL} should be able to get specific CRD in ${NAMESPACE}"
+	testPermissions "get" "crd/installations.installer.kyma-project.io" "${NAMESPACE}" "yes"
 
-	echo "--> developer@kyma.cx should be able to list Deployments in production"
-	testPermissions "developer@kyma.cx" "list" "clusterrole" "production" "yes"
+	echo "--> ${DEVELOPER_EMAIL} should NOT be able to delete ClusterRole in ${NAMESPACE}"
+	testPermissions "delete" "clusterrole" "${NAMESPACE}" "no"
 
-	echo "--> developer@kyma.cx should NOT be able to create Services in production"
-	testPermissions "developer@kyma.cx" "create" "service" "production" "no"
+	echo "--> ${DEVELOPER_EMAIL} should be able to list Deployments in production"
+	testPermissions "list" "clusterrole" "production" "yes"
 
-	echo "--> admin@kyma.cx should be able to get ClusterRole"
-	testPermissions "admin@kyma.cx" "get" "clusterrole" "${NAMESPACE}" "yes"
+	echo "--> ${DEVELOPER_EMAIL} should NOT be able to create Services in production"
+	testPermissions "create" "service" "production" "no"
 
-	echo "--> admin@kyma.cx should be able to delete Deployments"
-	testPermissions "admin@kyma.cx" "delete" "deploy" "${NAMESPACE}" "yes"
+    EMAIL=${ADMIN_EMAIL} PASSWORD=${ADMIN_PASSWORD} getConfigFile
+    export KUBECONFIG="${PWD}/kubeconfig"
```

Comment:
I wouldn't say we have to export that env var for each user, as it is set to the same value.